constants/faqs.ts

```diff
@@ -24,38 +24,62 @@ export const CXC_FAQ: QandA[] = [
 export const GENERAL_FAQ: QandA[] = [
```

Comment:
- [ ] Content seems good, but the FAQ code doesn't seem to take in the line breaks. Maybe try adding \n or some other way to have it show properly? I also uncommented the faq section on the homepage so you can look and see the result